Feature/poetry inclusion

K3llar opened the pull request:
Install poetry lib to project.
Create list of requirements (pyproject.toml) from actual requirements.txt. Filled require field such us description, license, homepage etc.
Added "How to use Poetry" chapter in README.md

K3llar committed:
create poetry files with dependencies from requirements.txt

K3llar committed:
update readme.md with instructions of using poetry lib

K3llar committed:
Update README.md

edit markdown syntaxis for poetry chapter

K3llar committed:
Update README.md

add request for team to update list of authors

AntonZelinsky reviewed (approved):
Нужно слить к себе ветку девелоп, конфликты пофиксить и сделать ребейз. Потом, можно будет мержить

K3llar committed:
create hook for export dev and prod requirements.
edit readme.md
edit .pre-commit-config.yaml

K3llar commented:
Немного запутался и вместо rebase сделал merge из develop в свою ветку. Можно будет сжать историю коммитов?
По последнему коммиту: добавил хук для экспорта зависимостей в папку requirements (мне кажется создание папки это хорошая идея чтобы не засорять корень проекта). Поправил порядок запуска в readme.md.

K3llar committed:
rebase with develop branch
remove pre-commit from production requirements with update pyproject and poetry.lock

K3llar commented:
сделал нормальный rebase с веткой develop - убрал ненужные коммиты других участников. Поправил файл pyproject.toml - убрал pre-commit из зависимостей для деплоя.

AntonZelinsky commented on pyproject.toml:
```diff
@@ -0,0 +1,63 @@
+description = "Service bot for doctors in \"Prosto sprosit\" team."
+readme = "README.md"
+authors = [
+  "Alexander Butakov <asbutakov@gmail.com>",
```
Ой, это точно лучше от сюда убрать. С одной стороны, мы пакеты делать не будем, с другой стороны, нужно тогда уже всех вносить. В общем, убираем

K3llar replied:
В файле readme я просил всех добавить себя. Но могу создать опросник в слаке чтобы все сбросили свою информацию

K3llar committed:
edit after review
requirements files now located at the root of the project
edit pyproject.toml authors chapter
